### PR TITLE
Add qemu-arm-static when chroot on non-arm platform

### PR DESCRIPTION
**Pull request by VertexC**
The image boots on raspberry3B model while failed on raspberry3+B.

### CODE REVIEW

File: bin/create_image.sh [hunk 1]
```diff
@@ -68,13 +68,23 @@ cp -r "${DIR}/../../client" /mnt/tmp/usr/src/client >> "${LOG}" 2>&1
 echo "${3}" > /tmp/profile.json
 mv /tmp/profile.json /mnt/tmp/usr/src/client/honeypot_profile.json
 cp -r "${DIR}/../pipot/services" /mnt/tmp/usr/src/client/pipot >> "${LOG}" 2>&1
+cpuArch=$(python ${DIR}/getCpuArch.py)
```
Comment (canihavesomecoffee) [inline bin/create_image.sh]: I don't see a `getCpuArch.py` file. Did you forget to add it, or am I overlooking something?

File: install/install.sh [hunk 2]
```diff
@@ -95,7 +95,7 @@ fi
 server_ip=`dig +short myip.opendns.com @resolver1.opendns.com`
 read -e -p "Server IP: " -i "${server_ip}" config_server_ip
 read -e -p "Server interface (SSL): " -i "443" config_server_port
-read -e -p "Application root (if not a whole (sub)domain, enter the path. None if whole (sub)domain): " -i "None" config_application_root
+read -e -p "Application root (if not a whole (sub)domain, enter the path. None if whole (sub)domain): " -i "'/'" config_application_root
```
Comment (canihavesomecoffee) [inline install/install.sh]: I thought (at least under the Flask version I used) that it should be `None`, as we'd use a full domain (e.g. `canihavesome.coffee`) and not a subdirectory... Did it change?

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]

File: install/install.sh (hunk 2 above)
Comment (VertexC) [reply install/install.sh]: The web page continues to jump back to `auth/login` if the `application_root` is `None`. Before I find any other supportive material, let's leave that to `None`.

File: bin/create_image.sh (hunk 1 above)
Comment (VertexC) [reply bin/create_image.sh]: Oh right, thanks for the double check.

Comment (canihavesomecoffee) [inline bin/create_image.sh]: Is there a reason why you use python instead of directly calling it here in bash?

**Comment by VertexC** [comment]
Close as PR #24 add the same feature.

**Pull request closed without merging**